eisbuk-admin/src/components/atoms/BirthdayMenu/BirthdayMenu.tsx

```diff
@@ -37,6 +38,17 @@ const BirthdayMenu: React.FC<Props> = ({ customers, onClickShowAll }) => {
     setBirthdaysAnchorEl(null);
   };
 
+  const calculateTimeDiff =
+    DateTime.now().plus({ day: 1 }).startOf("day").toMillis() -
+    DateTime.now().toMillis();
+  const [timeDiff, setTimeDiff] = useState(calculateTimeDiff);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setTimeDiff(calculateTimeDiff);
```

Comment:
Actually, `setTimeDiff would work better as a funtion. Here it needs to be invoked.